impl/src/main/java/slack/SlackResourceImpl.java

```diff
@@ -78,4 +83,12 @@ public SlackResourceImpl(SlackConfig slackConfig) {
                 throw new WebException(HttpResponseStatus.INTERNAL_SERVER_ERROR, channelsRepliesResponse.getError() + " " + channelsRepliesResponse.getWarning());
             });
     }
+
+    @Override
+    public Observable<Auth> getAuth(Message mainMessage) {
```

Comment:
Since only the user part of the message is required, why not just pass the user? The signature "getAuth(message)" doesn't explain how an auth is resolved from a message as passing a user object would :)